examples/scales/logit_demo.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
The rendered example is at https://22432-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/scales/logit_demo.html#sphx-glr-gallery-scales-logit-demo-py; you may want to play with e.g. figsize a bit...
Also, looking at the example I wonder whether $\frac{1}{2}$ is that nice, perhaps 1/2 would look better?

Reply:
yes. There is a way to build the html locally?

For one half, I prefer $\frac{1}{2}$, but it is a personal pov, I can put the choice in a option.

Reply:
Yes, see https://matplotlib.org/devel/documenting_mpl.html#building-the-docs.
I'll let the next reviewer (we need two to merge) decide on whether to insist on an API for 1/2 vs. \frac...

Reply:
I think both choice should be proposed, but we still have to decide the default choice.

Reply:
It's your PR, you get to pick, unless the other reviewer agrees with me that 1/2 is better :)

Reply:
I have added a option to choose the string for one half. It is illustrated in the example. I can change the default choice.